static/js/github-create-branch.js

```diff
@@ -81,6 +81,10 @@ $(document).ready(() => {
     data: []
   });
 
+	$("#ghParentBranch").on("change", function (e) {
+		validateSourceBranch(e.val);
```

Comment:
Originally only intended to test "manual" branch case
Now we check every time. 1 - its simpler. 2 - (pretty edgy) It catches the case if people take a long time between selecting branhes. 3 - It uses the User token, so if there is any mismatch it will be picked up here early and clearer error message